### PR TITLE
Volnag python

### PR DESCRIPTION
**Pull request by volnagPython**
Added two files (PT17 and modified Tk_OWM) from HW09

### CODE REVIEW

File: hw/hw10/volnagPython/volnagPython_PT19.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    
+       def __init__(self,shape,side1,side2): #Initializing a class
+           self.shape = shape
+           self.side1 = side1
+           self.side2 = side2
+
+       def __str__(self):
+           return f"{self.shape} with two sides "
+
+       def area(self):
```
Comment (lhalam) [inline hw/hw10/volnagPython/volnagPython_PT19.py:39]: use return, not print

Comment (volnagPython) [reply hw/hw10/volnagPython/volnagPython_PT19.py:42]: Done.

File: hw/hw10/volnagPython/volnagPython_PT20.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+    def __init__(self,name,spieces):
+        self.name = name
+        self.spieces = spieces
+
+    def __str__(self):
+        return f"This is a {self.spieces} named {self.name}."
+        
+    def greets(self):
+        print(f"Hello {self.name}!")
```
[the remark this answers is not in the document]
Comment (volnagPython) [reply hw/hw10/volnagPython/volnagPython_PT20.py]: Done.

File: hw/hw10/volnagPython/volnagPython_PT20.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+    def message(self):
+        print(f"How are you doing {self.name.upper()}?")
```
[the remark this answers is not in the document]
Comment (volnagPython) [reply hw/hw10/volnagPython/volnagPython_PT20.py]: Done.

**Pull request merged**